task/02-numbers-tasks.js

```diff
@@ -221,20 +233,23 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+  if (!Number(value)) {
+    return def;
+  }
+  return Number(value);
 }
```

Comment:
not correct, what if value = '0' ?
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN